Packs/Gamma/Integrations/Gamma/Gamma.py

```diff
@@ -0,0 +1,321 @@
+
+    violation = client.get_violation(violation_id)
+
+    if violation['response'][0]['violation_id'] != int(violation_id):
+        return "Violation with this ID does not exist."
+
+    human_readable = ''
+    for i in violation['response']:
+        human_readable += f'### Violation {i["violation_id"]} \r' \
+                          f'|Violation ID|Status|Timestamp|Dashboard URL|User|App Name| \r' \
+                          f'|---|---|---|---|---|---| \r' \
+                          f'| {i["violation_id"]} | {i["violation_status"]} | {timestamp_to_datestring(i["violation_event_timestamp"] * 1000)} | {i["dashboard_url"]} | {i["user"]} | {i["app_name"]} | \r'
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix="GammaViolation",
+        outputs_key_field="violation_id",
+        outputs=violation,
+        raw_response=violation
+    )
+
+
+def update_violation_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    :type client: Client
+    :param client: Gamma client
+
+    :param args: all command arguments, usually passed from demisto.args()
+        args['name'] is used as input name
+
+    :return:
+        A CommandResults object that is then passed to return_results
+
+    :rtype: ``CommandResults``
+    """
+
+    violation = args["violation"]
+    status = args["status"].upper()
+    notes = args["notes"]
+
+    if not int(violation) >= 1:
+        raise ValueError("violation must be greater than 0")
+    if status not in VALID_VIOLATION_STATUSES:
+        raise ValueError("status must be one of the following: OPEN, RESOLVED, IGNORED")
+
+    client.update_violation(violation, status, notes)
+
+    updated_violation = client.get_violation(violation)
+    human_readable = ''
+    for i in updated_violation['response']:
+        violation_id = i['violation_id']
+        human_readable += f'### Updated Violation {i["violation_id"]} \r' \
+                          f'|Violation ID|Status|Timestamp|Dashboard URL|User|App Name| \r' \
+                          f'|---|---|---|---|---|---| \r' \
+                          f'| {violation_id} | {i["violation_status"]} | {timestamp_to_datestring(i["violation_event_timestamp"] * 1000)} | {i["dashboard_url"]} | {i["user"]} | {i["app_name"]} | \r'
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix="GammaViolation",
+        outputs_key_field="violation_id",
+        outputs=updated_violation,
+        raw_response=updated_violation
+    )
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    api_key = demisto.params()['api_key']
+
+    # get the service API url
+    base_url = urljoin(demisto.params()['url'], '/api/discovery/v1/')
+
+    verify_certificate = not(demisto.params().get('insecure', False))
+    proxy = demisto.params().get('proxy', False)
```

Comment:
Those parameters are not defined in the `configuration` section in the YML file.